packages/sdk/contractkit/src/wrappers/StableTokenRegistry.test.ts

```diff
@@ -0,0 +1,33 @@
+import { Address } from '@celo/base/lib/address'
+import { testWithGanache } from '@celo/dev-utils/lib/ganache-test'
+import Web3 from 'web3'
+import { newKitFromWeb3 } from '../kit'
+import { StableTokenRegistryWrapper } from './StableTokenRegistry'
+
+testWithGanache('StableTokenRegistry Wrapper', (web3: Web3) => {
+  const kit = newKitFromWeb3(web3)
+  let accounts: Address[] = []
+  let stableTokenRegistry: StableTokenRegistryWrapper
+
+  beforeAll(async () => {
+    accounts = await web3.eth.getAccounts()
+    kit.defaultAccount = accounts[0]
+    stableTokenRegistry = await kit.contracts.getStableTokenRegistry()
+  })
+
+  describe('Verifying that it always has correct values', () => {
+    it('has the correct list of fiatTicker', () => {
+      expect(stableTokenRegistry.FiatTickers()).toEqual(['USD', 'EUR', 'BRL'])
```

Comment:
careful with the capitalization

Reply:
values I've hardcoded in the contract are also capitalised, but I think it's worth writing a note about it